src/zidx.c

```diff
@@ -1370,6 +1454,11 @@ int zidx_fill_checkpoint(zidx_index* index,
     /* dict_length can't be more than 32768. */
     new_checkpoint->window_length = dict_length;
 
+    /*compute checksum*/
+    uint32_t new_checksum=crc32(0L,Z_NULL,0);
+    new_checksum=crc32_z(new_checksum,new_checkpoint->window_data,new_checkpoint->window_length);
```

Comment:
Linker complains about `crc32_z` on CI. Let's use `crc32` instead. Downcast length if compiler complains. Window length should always be <= 32KB.